core/webpack.config.js

```diff
@@ -1,13 +1,14 @@
 const { readFileSync } = require('fs');
-const babelSettings = JSON.parse(readFileSync('.babelrc'));
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const sass = require('node-sass');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 
+const babelSettings = JSON.parse(readFileSync('.babelrc'));
+const commonRules = require('./webpack-common-rules');
+const commonPlugins = require('./webpack-common-plugins');
+
 module.exports = {
   entry: {
-    index: [
+    luigi: [
       // './node_modules/fiori-fundamentals/dist/fiori-fundamentals.min.css',
```

Comment:
Guess we can remove the min version at all to not accidentally add duplicates?

Reply:
sure, done. Anyway, we will have in the future a check for bundlesizes in our CI. See #588